packages/service/src/main.ts

```diff
@@ -30,6 +31,7 @@ function setupRouting(): void {
   app.use('/commands/slack', SlackCommandController);
   app.use('/commands/text', TextCommandController);
   app.use('/x-sounds', XSoundsController);
+  app.use('/songs', SongController);
```

Comment:
```suggestion
  app.use('/songs', SongsController);
```
i nazwa pliku do zmiany
`packages/service/src/domain/songs/songs-controller.ts`

Reply:
To do zmiany też `song-repository` i `song-service`